cli/pkg/telemetry/telemetry.go

```diff
@@ -88,22 +93,24 @@ func (t *Telemetry) Flush(ctx context.Context) error {
 }
 
 type behaviourEventFields struct {
-	AppName       string `json:"app_name"`
-	InstallID     string `json:"install_id"`
-	BuildID       string `json:"build_id"`
-	Version       string `json:"version"`
-	UserID        string `json:"user_id"`
-	IsDev         bool   `json:"is_dev"`
-	Mode          string `json:"mode"`
-	Action        string `json:"action"`
-	Medium        string `json:"medium"`
-	Space         string `json:"space"`
-	ScreenName    string `json:"screen_name"`
-	EventDatetime int64  `json:"event_datetime"`
-	EventType     string `json:"event_type"`
+	AppName       string   `json:"app_name"`
+	InstallID     string   `json:"install_id"`
+	BuildID       string   `json:"build_id"`
+	Version       string   `json:"version"`
+	UserID        string   `json:"user_id"`
+	IsDev         bool     `json:"is_dev"`
+	Mode          string   `json:"mode"`
+	Action        string   `json:"action"`
+	Medium        string   `json:"medium"`
+	Space         string   `json:"space"`
+	ScreenName    string   `json:"screen_name"`
+	EventDatetime int64    `json:"event_datetime"`
+	EventType     string   `json:"event_type"`
+	SourceDriver  []string `json:"source_driver"`
+	OLAPDriver    []string `json:"olap_driver"`
```

Comment:
Can we use a generic `Payload map[string]any` or something like that instead? Unlike the other fields, these properties seem specific to a single event

Reply:
It depends on how druid parses nested multi-value dimensions
We can also mix in additional dimensions
```
    val additionalDims map[string]any

    // Convert struct to map and merge with additional data
    var resultMap map[string]interface{}
    temp, _ := json.Marshal(fields)
    json.Unmarshal(temp, &resultMap)
    for key, value := range additionalDims {
        resultMap[key] = value
    }

    // Marshal the combined data to JSON
    event, _ := json.Marshal(resultMap)
```

Reply:
isolated this into a separate commit